web-components/src/components/tooltip/Tooltip.ts

```diff
@@ -44,120 +45,83 @@ export namespace Tooltip {
 @customElement("md-tooltip")
 export class Tooltip extends FocusMixin(LitElement) {
   @property({ type: String }) message = "";
-  @property({ type: String }) placement: Tooltip.Placement = "auto";
-  @property({ type: Boolean }) disabled = false;
```

Comment:
@boiarchukk Why are you removing disabled? We really need this. Now all tooltips are enabled always.